src/modules/transaction/components/TransactionPriority/TransactionPriority.js

```diff
@@ -25,18 +22,18 @@ const TransactionPriority = ({
   composedFees,
   customFee,
   minRequiredBalance,
+  formProps,
+  computedMinimumFees,
 }) => {
   const [showEditIcon, setShowEditIcon] = useState(false);
-  const [inputValue, setInputValue] = useState();
-
-  const maxFee = MODULE_COMMANDS_MAP[moduleCommand].maxFee;
+  const [inputValue, setInputValue] = useState({});
```

Comment:
Why do inputValue need to be an object. With such a name i think of it as a string 🤔